tests/test_httpserver_dynamic.py

```diff
@@ -0,0 +1,87 @@
+import pytest
```

Comment:
This entire file was just my scratch file when trying to figure out how this whole dynamic httpserver stuff could work. I cleaned it up and commented for the sake of explaining the mechanics. It's not really testing anything in `poglink`, so I might suggest we remove it. But I'm leaving here for now so you can review a more simplified example than what you see in the actual rates cog test.

Reply:
Much appreciated, I looked back on this a few times to figure out what was going on in the rates cog test.